ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/events/ContractsReader.scala

```diff
@@ -127,7 +227,7 @@ private[appendonlydao] sealed class ContractsReader(
   }
 
   /** Lookup of a contract in the case the contract value is already known (loaded from a cache) */
-  private def lookupActiveContractWithCachedArgument(
+  override def lookupActiveContractWithCachedArgument(
```

Comment:
set of metrics used by this method is different between `dao` and `appendonly`

Reply:
Done.